### PR TITLE
Fix entry action search in saiapi.cpp.j2

### PR DESCRIPTION
**Pull request by marian-pritsak**
The refactoring introduced in https://github.com/sonic-net/DASH/pull/438 only looks for a proper action attribute in SAI objects, not entries. In case of SAI entry, it iterates through all the attributes and matches the values regardless of attribute ID.

This PR aligns action search for both objects and entries, and adds action ID attributes to missing create_entry calls in tests.

### CODE REVIEW

**Comment by r12f** [comment]
hi Marian, do you mind to summary a bit on the background of this PR?

**Comment by marian-pritsak** [comment]
> hi Marian, do you mind to summary a bit on the background of this PR?

Done.

**Review by r12f — APPROVED** [review]
lgtm! thanks Marian!

File: dash-pipeline/SAI/templates/saiapi.cpp.j2 [hunk 1]
```diff
@@ -443,22 +443,25 @@ static sai_status_t dash_sai_create_{{ table.name }}(
     // Search the action
     for (uint32_t i = 0; i < attr_count; i++)
     {
-        switch(attr_list[i].value.s32)
+        if (SAI_{{ table.name | upper }}_ATTR_ACTION == attr_list[i].id)
```
Comment (kcudnik) [inline dash-pipeline/SAI/templates/saiapi.cpp.j2:446]: previously there was switch, and now there is if, and if attribure is not action, then all other attributes are skipped, in previous case, at least it was logged that some attribue is passed, now there is no log message, 

i propose to make this (if) and then at the end make continue, and leave switch statement as is after if.ACTION case

**Pull request merged**